ports/stm32f4xx/dlmstp.c

```diff
@@ -175,6 +175,12 @@ static unsigned ReceivePDUCount;
 static volatile struct dlmstp_packet PDU_Buffer[MSTP_PDU_PACKET_COUNT];
 static RING_BUFFER PDU_Queue;
 
+/* Callback function to be called every time we receive a preamble */
+static dlmstp_hook_frame_rx_start_cb PreamCallback = NULL;
+
+/* Callback function to be called every time we receive a frame */
+static dlmstp_hook_frame_rx_compl_cb FrameRxCallback = NULL;
```

Comment:
please rename `dlmstp_hook_frame_rx_compl_cb` to `dlmstp_hook_frame_rx_complete_cb`
Abbreviations may unintentionally encrypt the words and not be understandable by others.

Reply:
Fixed